Source/OnlineSubsystemEpic/Private/OnlineUserInterfaceEpic.cpp

```diff
@@ -710,6 +729,9 @@ TSharedPtr<FOnlineUser> FOnlineUserEpic::GetUserInfo(int32 LocalUserNum, const c
 		{
 			FUniqueNetIdEpic const epicUserId = static_cast<FUniqueNetIdEpic const>(UserId);
 
+			UE_LOG_ONLINE_USER(Log, TEXT("Local: %s:%s"), __FUNCTIONW__, *localUserId->ToDebugString());
```

Comment:
`__FUNCTIONW__` is a nonstandard addition to the compiler by Microsoft, use `__FUNCTION__` instead